python/nano/src/bigdl/nano/pytorch/patching/gpu_cpu/gpu_cpu.py

```diff
@@ -185,6 +199,20 @@ def patch_cuda(disable_jit=True):
 
 
 def unpatch_cuda():
+    '''
+    unpatch_cuda is an reverse function to patch_cuda. It will change the application
+    back to be available on cuda.
+
+    e.g.
+        >>> from bigdl.nano.pytorch.patching import unpatch_cuda
+        >>> unpatch_cuda()  # be sure it is used after patch_cuda
```

Comment:
in fact, `unpatch_cuda()` can be called before `patch_cuda`, it will do nothing in this case